test/unit/error_events.test.js

```diff
@@ -56,24 +64,34 @@ describe('Error events', function() {
         const timestamp = 0
         const exception = new Exception({error, customAttributes, timestamp})
         tx.addException(exception)
+
+        const segment = tx.agent.tracer.getSegment()
+
+        const spanAttributes = segment.attributes.get(DESTINATIONS.SPAN_EVENT)
```

Comment:
ideally, we'd prob actually verify on span since we'll be ending the transaction anyways but tis miinor.